### PR TITLE
[FE][SYNC-230] Topic Recommend Section

### PR DESCRIPTION
**Pull request by david20571015**
## Changes:
- Use `sass` instead of [deprecated](https://sass-lang.com/blog/libsass-is-deprecated) `node-sass`.
- Add topic recommend section in the homepage.

## Note
- This branch is developed base on unmerged pull request #64.

## Todo
- Get recommend hashtags by api.

## Screenshots (optional)
image

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -57,7 +57,7 @@
     "eslint-plugin-vue": "^6.2.2",
     "jest": "^27.0.5",
     "nock": "^13.2.7",
-    "node-sass": "^4.14.1",
+    "sass": "^1.55.0",
```
Comment (yccodr) [inline package.json:60]: Has been replaced in #63. Please revert this commit.

File: src/views/Home.vue [hunk 2]
```diff
@@ -253,6 +278,17 @@ export default {
         this.pickedHeadline %= Math.min(this.headline.length, 5)
       }, this.HEADLINEINTERVAL)
       this.pickedHeadline = 0
+    },
+    getLoginStatus() {
+      return this.$store.getters.isLogin
+    },
```
Comment (yccodr) [inline src/views/Home.vue]: `this.$store.getters.isLogin` can be assigned as a computed property. See https://vuejs.org/guide/essentials/computed.html#basic-example and https://v3.vuex.vuejs.org/guide/getters.html#the-mapgetters-helper

File: src/views/Home.vue [hunk 3]
```diff
@@ -253,6 +278,17 @@ export default {
...
+    getRecommendHashtags() {
+      // TODO: get recommend hashtags by api
+      return [
+        '九合一大選', '強尼戴普', '加密貨幣', 'COVID-19', '墮胎自主權',
+        '防疫政策', '烏俄戰爭', '加密貨幣', '普丁', '快篩試劑',
+        '兒童疫苗', '金曲獎33', '猴痘', '確診'
+      ]
```
Comment (yccodr) [inline src/views/Home.vue]: Write a mock API. This component will not have to be changed later.

**Review by newb1er — CHANGES_REQUESTED** [review]

File: src/views/Home.vue (hunk 2 above)
Comment (david20571015) [reply src/views/Home.vue]: Done.

File: src/views/Home.vue (hunk 3 above)
Comment (david20571015) [reply src/views/Home.vue]: Done.

**Comment by yccodr** [comment]
One last thing. Maybe we can wrap the whole section into a component? Since we are going to refactor the whole home page, it could be a disaster if anyone changed your code.

**Comment by david20571015** [comment]
> One last thing. Maybe we can wrap the whole section into a component? Since we are going to refactor the whole home page, it could be a disaster if anyone changed your code.

Agree. I'll refactor it later.

**Comment by david20571015** [comment]
> One last thing. Maybe we can wrap the whole section into a component? Since we are going to refactor the whole home page, it could be a disaster if anyone changed your code.

Done.

**Review by newb1er — APPROVED** [review]
looks good to me

**Pull request merged**